x-pack/plugins/maps/public/store/map.js

```diff
@@ -84,6 +84,7 @@ const updateLayerSourceDescriptorProp = (state, layerId, propName, value) => {
 const INITIAL_STATE = {
   ready: false,
   goto: null,
+  tooltipState: null,
```

Comment:
This needs to default to empty object otherwise `clearTooltipStateForLayer` has a bug were it reads `tooltipState.layerId` from null.

To view, open a new map and toggle tile layer visibility

<img width="350" alt="screen shot 2019-03-06 at 8 43 11 am" src="https://user-images.githubusercontent.com/373691/53893685-e6968780-3feb-11e9-8ac3-d705f00065bd.png">

Reply:
Actually, the better fix is to update `clearTooltipStateForLayer` to handle null value for `tooltipState` since tooltipState will be set to null when cleared